### ISSUE TITLE
final certificate does not contain subject alt names (SAN)

### ISSUE THREAD

**Issue by mbruzek** [juju-solutions/layer-tls#1]
The request contains the SANs but the final certificate does not.

The `easyrsa sign-req` operation needs the --subject-alt-names flag, but the server does not have this information (public, private, hostname of the remote unit).  Need to send this SANs string over the relation or some other way.  The SAN is correctly in the remote unit, perhaps there is a way to get the information out of there.

Helpful commands:
```bash
openssl x509 in pki/issued/tls_1.crt -text
openssl req -in pki/reqs/ubuntu_1.req -text
```

**Comment by chuckbutler** [comment]
related to: https://github.com/OpenVPN/easy-rsa/issues/60

**Comment by chuckbutler** [comment]
Closed with #26

**Comment by mbruzek** [comment]
You are correct. Thanks for fixing this!
